### PR TITLE
Updated to OS X 10.7 Lion.

### PR DESCRIPTION
**Pull request by FluxAugur**
Updated project to Xcode 4.3 standards.
Corrected/removed invalid element scaling property references (or reset
to the default values).
Corrected/removed deprecated member references throughout.
Corrected/removed deprecated method references (mostly relating to
NSOpenPanel).
Rewrote inRangeScriptChange and outOfRangeScriptChange to function in
lieu of deprecated methods.

### CODE REVIEW

File: AppController.m [hunk 1]
```diff
@@ -306,8 +306,8 @@ - (IBAction)changeDevice:(id)sender
 	[device retain];
 	
 	[deviceName setStringValue:[NSString stringWithFormat:@"%@ (%@)",
-								[device getName],
-								[device getAddressString]]];    
+								[device name],
+								[device addressString]]];    
 }
```
Comment (FluxAugur) [inline AppController.m:311]: Deprecated as of OS X 10.6. Simply updated naming.

File: AppController.m [hunk 2]
```diff
@@ -355,10 +355,17 @@ - (IBAction)enableMonitoring:(id)sender
...
+  fileTypesArray = [NSArray arrayWithObjects:@"scpt", nil];
+  
+  [op setCanChooseFiles:YES];
+  [op setAllowedFileTypes:fileTypesArray];
+  
+  if ( [op runModal] == NSOKButton ) {
+    NSArray *filenames = [op URLs];    
+  	[inRangeScriptPath setStringValue:[[filenames objectAtIndex:0] path]];
+  }
 }
```
Comment (FluxAugur) [inline AppController.m:370]: Deprecated as of OS X 10.6. Rewritten to use currently supported methods. Minor error-checking added. Logic unchanged.

File: AppController.m [hunk 3]
```diff
@@ -374,10 +381,17 @@ - (IBAction)inRangeScriptTest:(id)sender
...
+  fileTypesArray = [NSArray arrayWithObjects:@"scpt", nil];
+  
+  [op setCanChooseFiles:YES];
+  [op setAllowedFileTypes:fileTypesArray];
+  
+  if ( [op runModal] == NSOKButton ) {
+    NSArray *filenames = [op URLs];    
+  	[outOfRangeScriptPath setStringValue:[[filenames objectAtIndex:0] path]];
+  }
 }
```
Comment (FluxAugur) [inline AppController.m:395]: Deprecated as of OS X 10.6. Rewritten to use currently supported methods. Minor error-checking added. Logic unchanged.

File: Info.plist [hunk 4]
```diff
@@ -13,22 +15,22 @@
...
 	<string>MainMenu</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
-	<key>LSUIElement</key>
-	<true/>
-	<key>CFBundleShortVersionString</key>
-	<string>1.5</string>
-	<key>CFBundleGetInfoString</key>
-	<string>1.5</string>
 </dict>
```
Comment (FluxAugur) [inline Info.plist:35]: Updated version and other information in new Info.plist.

**Comment by Daij-Djan** [comment]
would the renaming not break backward compatibility?
IMHO

**Pull request merged**